index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" href="./css/style.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <section class="header">
+        <h2>My Logo</h2>
+        <i class="material-icons" >	list</i>
+    </section>
+    <section class="container">
+    
+        <h1><b>I’m Jalal.<br>
+
+            Glad to see<br>
+            you!</b></h1><br>
+        <p>I’m a software developer! I can help youbuild a product , feature or website Lookthrough some of my work and
+            experience! If you like what you see and
+            have a project you need coded, don’t
+            hesitate to contact me.<br><br><span>LETS CONNECT</span></p>
+         <div class="icons">
+            <img class="icon1" src="Vector (10).jpg" alt="">
+            <img class="icon2" src="Vector (8).jpg" alt="">
+            <img class="icon3" src="Vector (9).jpg" alt="">
+            <img class="icon4" src="Vector (7).jpg" alt="">
+            <img class="icon5" src="Vector (11).jpg" alt="">
+            <img class="icon6" src="Vector (4).jpg" alt="">
+            <img class="icon7" src="Vector (5).jpg" alt="">
+            
+        </div>
```

Comment:
- [ ] Please make sure that the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href` ⛏✨ You can consider using something like below 👇


```
<ul>
  <li>
    <a href="replace it with the appropriate link" target="_blank">
      <img  src="replace it with the appropriate path" alt="GitHub">
    </a>
  </li>
  ...
</ul>
```